pkg/sloop/webserver/bindata.go

```diff
@@ -15,7 +15,7 @@
 // webfiles/sloop_ui.js
 // DO NOT EDIT!
 
-package webserver
```

Comment:
Why do we have the package name change?

Reply:
I have no idea why its changed, all I did is to run 'go-bindata -o bindata.go webfiles/' within the webserver directory. Can I manually change/update the package name in bindata.go file? Or I should run the command again?

Reply:
@sana-jawad Manually change package name back to webserver